Tests/JobsTests/JobsTests.swift

```diff
@@ -78,6 +78,32 @@ final class JobsTests: XCTestCase {
             .hourly()
             .at(30)
     }
+    
+    func testRepeatingScheduledJob() throws {
+        let app = Application(.testing)
+        defer { app.shutdown() }
+        
+        XCTAssertEqual(TestingScheduledJob.count, 0)
+        app.jobs.schedule(TestingScheduledJob()).everySecond()
```

Comment:
This should be producing a warning, right?

Reply:
Nope. Why?

Reply:
Hmm weird, I'll look into this.

Reply:
Oh wait, this is fixed now. The PR body confused me. I understand now haha.

Reply:
Oh ok cool! :)

Reply:
Is that related to this issue? 
https://github.com/apple/swift-nio/issues/1334